resources/views/pages/user_profile.blade.php

```diff
@@ -0,0 +1,42 @@
+@extends('layouts.main_layout')
+
+@section('head')
+
+    <title>user_profile</title>
+
+@endsection
+
+@section('mainBody')
+
```

Comment:
This class needs some serious formatting love. Align the elements correctly and get rid of extra white space.